src/components/createTestCase/CreateTestCase.tsx

```diff
@@ -143,6 +146,8 @@ const CreateTestCase = () => {
   const [showValidationErrors, setShowValidationErrors] = useState(false);
   const [measurementPeriodStart, setMeasurementPeriodStart] = useState<Date>();
   const [measurementPeriodEnd, setMeasurementPeriodEnd] = useState<Date>();
+  const [changedId, setChangedId] = useState("");
```

Comment:
```suggestion
  const [changedPopulation, setChangedPopulation] = useState("");
```
I had to deep dive down into the trigger change function to figure out Id === population. Suggest renaming.

Reply:
Sure, I'll change it